### DISCUSSION TITLE
How to pass input and get output directly without gradio_client, on Google Colab

### DISCUSSION THREAD

**Discussion by zion4343** [lllyasviel/Fooocus#3601] [General]
I would like to ask about how to pass input and get output directly without API on Google colab.

Could I use Fooocus without gradio API?
Also, then, can I set specific setting for the output?

Please let me know how to use Fooocus without gradio API.

**Comment by IPv6** [comment]
There are forks that support FastAPI -  https://github.com/mrhan1993/Fooocus-API and others, google for it.
Fooocus itself have no convenient api, it is primarily UI tool
